code/modules/food_and_drink/cakes.dm

```diff
@@ -208,9 +636,9 @@
 /obj/item/cake_item/attack(target as mob, mob/user as mob)
 	var/iteminside = src.contents.len
 	if(!iteminside)
-		boutput(user, "<span class='alert'>The cake crumbles away!</span>")
+		user.show_text("The cake crumbles away!","red")
 		qdel(src)
-	boutput(user, "<span class='notice'>You bite down on something odd! You open up the cake...</span>")
+	user.show_text("You bite down on something odd! You open up the cake...","red")
```

Comment:
🤔

Reply:
i didn't make any of these boutputs and they're not here client-side either! D:

Reply:
![Code_cBOHXLe0vS](https://user-images.githubusercontent.com/12144569/104827912-d8cd0280-580f-11eb-9ebc-4950e22a357b.png)

Reply:
unless i was the one that originally made them show_text in the first place and i don't remember it....which is possible. it's been almost 6 months since i worked on this last...either way show_text plays nice with dark mode

Reply:
no?